src/Runtime/python/PyExecutionSessionBase.hpp

```diff
@@ -36,7 +36,9 @@ class PyExecutionSessionBase
       bool defaultEntryPoint = true);
   std::vector<std::string> pyQueryEntryPoints();
   void pySetEntryPoint(std::string entryPointName);
-  std::vector<py::array> pyRun(const std::vector<py::array> &inputsPyArray);
+  std::vector<py::array> pyRun(const std::vector<py::array> &inputsPyArray,
```

Comment:
Can you add a comment here on what are the expectations on the inputs? Like each array has to have the same number of entires... Also, I understand the array (data input) and the shape (size of the data). Can you add an explanation on the strides?

Reply:
Comments added. Thanks.